### PR TITLE
Fix electrum query failing (CloseNotify bug)

### PR DESCRIPTION
**Pull request by devbordecraft**
Retry query the electrum server if it fail. This address #797

### CODE REVIEW

**Comment by binarybaron** [comment]
Great, thanks for taking the initiative to fix this. I think it might be useful to have a list of Electrum nodes and fall back to another server if the primary one fails.

File: swap/src/asb/config.rs [hunk 1]
```diff
@@ -58,7 +58,7 @@ impl GetDefaults for Mainnet {
             data_dir: default_asb_data_dir()?.join("mainnet"),
             listen_address_tcp: Multiaddr::from_str("/ip4/0.0.0.0/tcp/9939")?,
             listen_address_ws: Multiaddr::from_str("/ip4/0.0.0.0/tcp/9940/ws")?,
-            electrum_rpc_url: Url::parse("ssl://electrum.blockstream.info:50002")?,
+            electrum_rpc_url: Url::parse("ssl://blockstream.info:700")?,
```
Comment (rishflab) [inline swap/src/asb/config.rs:61]: Does the old url no longer work?

**Review by rishflab — APPROVED** [review]

File: swap/src/asb/config.rs (hunk 1 above)
Comment (devbordecraft) [reply swap/src/asb/config.rs:61]: It work but it is very unstable. That's why I propose to change to this electrum node referenced in the official blog of Blockstream : https://blog.blockstream.com/en-esplora-and-other-alternatives-to-electrumx/

**Comment by devbordecraft** [comment]
> Great, thanks for taking the initiative to fix this. I think it might be useful to have a list of Electrum nodes and fall back to another server if the primary one fails.

No problem ❤️, good idea, will make an issue for that

**Review by da-kami — APPROVED** [review]

**Pull request merged**